### PR TITLE
Blog post on collaboration networks

### PR DESCRIPTION
**Pull request by yabellini**

### CODE REVIEW

File: content/blog/2024-06-25-collaboration-networks/index.en.md [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+We can also calculate the degree of each node, the betweenness and the closeness to analize some of the characteristics of the nodes and the network.
+
+``` r
+degree(g_blog)
+betweenness(g_blog)
+closeness(g_blog)
+```
+
+As we can identify network members we know that Maëlle Salmon have the maximun number of contributions as active staff member and Noam Ross have the higest degree and the higest centrality. It is the most connected member in the blog post network. 
+
```
Comment (steffilazerte) [inline content/blog/2024-06-25-collaboration-networks/index.en.md:244]: I think this section would be better on it's own, before the conclusion.

Also, I removed two points: 

* *Knowing the nodes help to understand the clusters and the interactions.*

I'm not sure there's any point in a community creating a network analysis if they don't know the nodes. Would this ever happen? I thought perhaps it wasn't likely so we don't need to include it?

 * *Lean on the people who have been in the network for the longest time.*

This is a bit different from the other tips. The other tips suggest what to do to create the network and conduct the analysis. This point seems related to what to do with the results. If you want to include it, I would rephrase it: 

- Use your network to better understand and guide your community 
    - Leverage the knowledge and experience of members who have been in the network for a long time
    - Ask well connected community members to help promote collaborations among other clusters
    - *etc.*

I prefer using the idea of "leveraging on the people who have been in the network" instead of "leaning". "Leveraging" implies "making good use of valuable resources" whereas "leaning" implies letting others do the work. This is appropriate for friends, family and colleagues, i.e. lean on your friends in times of hardship; but less so for community members or volunteers (depending on the task). 

```suggestion
## Using Social Network Analysis in your community

Social Network Analysis is a powerful tool for understanding interactions and collaborations in a community and there are powerful R packages to help with the analysis. So, what if you wanted to do the same for your community? Here are our tips.

* Think about what the nodes in your community are (people, countries, organizations, ...?) 
* Think about the type(s) of connection you have in your community.
    * Start with your paths for contributions.
    * Identify which contributions can be done in teams.
* Consider that you might already have information about these connections.
* Take into account open/closed data and any privacy considerations.
* Automate what you can in the data collection.
    * Formalize the workflow (code ;-)) so you can repeat & reproduce.
* Remember that it is hard to capture all type of interactions.
* You can take snapshot of the network model ... 
    * ... so you can compare it at different times.
    * ... so you can use it for measure the impact of interventions and programs.
* Share what you find with your community
    * ... and other community managers!
```

File: content/blog/2024-06-25-collaboration-networks/index.en.md [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+  - Yanina Bellini Saibene
+  - Sandro Camargo
+editor:
+  - Steffi LaZerte
+date: '2024-06-25'
+slug: collaboration-networks
+categories:
+  - blog
+tags:
+  - Community
```
Comment (steffilazerte) [inline content/blog/2024-06-25-collaboration-networks/index.en.md:13]:
```suggestion
  - Community
  - networks
  - social network analysis
  - community-manager-tools
```

File: content/blog/2024-06-25-collaboration-networks/index.en.md [hunk 3]
```diff
@@ -0,0 +1,267 @@
+---
+title: "rOpenSci Community Collaboration Insight Using Social Networks Analysis"
+author: 
+  - Yanina Bellini Saibene
+  - Sandro Camargo
+editor:
+  - Steffi LaZerte
+date: '2024-06-25'
```
Comment (steffilazerte) [inline content/blog/2024-06-25-collaboration-networks/index.en.md:8]: This is a very cool post, adjust the date, but I wouldn't try to squish it in anywhere. This week has an American holiday on Thursday, and I think a lot of Americans are taking a long weekend. I would publish next Tuesday as a good day to gather attention (not Monday when everyone's playing catch up). But then when I look at the [list of international Holidays](https://www.timeanddate.com/holidays/), it looks like Argentinian Independence day is Tuesday? So perhaps Wednesday or Thursday next week?

File: content/blog/2024-06-25-collaboration-networks/index.en.md
Comment (steffilazerte) [inline content/blog/2024-06-25-collaboration-networks/index.en.md:1]: Don't forget to change the date on the folder 😁

File: content/blog/2024-06-25-collaboration-networks/index.en.md [hunk 4]
```diff
@@ -0,0 +1,267 @@
...
+slug: collaboration-networks
+categories:
+  - blog
+tags:
+  - Community
+description:
```
Comment (steffilazerte) [inline content/blog/2024-06-25-collaboration-networks/index.en.md:14]: Don't forget to put in a description

File: content/blog/2024-06-25-collaboration-networks/index.en.md [hunk 5]
```diff
@@ -0,0 +1,267 @@
...
+editor:
+  - Steffi LaZerte
+date: '2024-06-25'
+slug: collaboration-networks
+categories:
+  - blog
+tags:
+  - Community
+description: 
+
```
Comment (steffilazerte) [inline content/blog/2024-06-25-collaboration-networks/index.en.md:15]: Do you want to `preface` this with a mention that this is based on/related to a talk you gave at useR (latinR?) in 2022? With a link to slides etc. if you have it.

**Review by steffilazerte — COMMENTED** [review]
Hi @yabellini, 

What a cool post! I really like it. Yes it is longish, but I don't think it's too long. It has several defined sections which are each valuable and not individually very long, so I'd leave it as is. 

Because this struck me as more than just your average blog post, I've been extra fastidious about structure and wording. In particular, I'm very picky about parallelisms. That is if you refer to 5 things in more than one place, they should always be in the same order. Similarly, if you have a list of steps, the structure of each step should be the same. 

I struggled a bit with the changes in the subject (I vs. we). In the end I think it's fine, but gave a couple of comments where I think it could be clarified a bit.

I also struggled with tenses, as sometimes it's something "we did" and sometimes it's something "you can do". Again, I think this is fine, but I added some comments to that I hope will help smooth the transitions. 

With the code, I'm picky about using `package::function()` consistently (at least within a block of code) and with being clear about `library(package)` calls where you don't use `package::function()` notation. 

My general suggestions (and these are in the comments), are to make sure you explain the network figures. Give some indications as to what they tell us and perhaps why that's interesting. There also a couple of network analysis terms that weren't defined and probably should be (for people like me, who think this is cool but have no idea how it works 😁).

I also suggest not rushing the publication. This is a very cool post! And I don't think it should be squeezed in at the end of the week (especially since the USA and all their associated territories have a holiday on Thursday, which means many will be taking a long weekend). So consider publishing next week. Also, consider marking it as a 'featured' blog post.

Thanks! This was a fun read!

* [X] post follows Content Guidelines
* [X] post follows Style Guide **[Once my edits are accepted 😉 ]**
* [X] title is in Title Case
* [x] publication date is ok 
* [X] alternative text of images is informative
* [ ] author metadata is provided with correct folder name
* [X] html not included in pull request of Rmd post
* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md
* [X] YAML subject tags are ok ("tech notes" for tech notes; "community" for non-staff non-editor)
* [ ] YAML field 'preface' is present if necessary **[Do you want to preface this with a mention that this is based on/related to a talk you gave at useR (latinR?) in 2022? With a link to slides etc. if you have it.]**

File: content/blog/2024-06-25-collaboration-networks/index.en.md
Comment (yabellini) [reply content/blog/2024-06-25-collaboration-networks/index.en.md:1]: Yes, I agree with you; we can publish next week or the other.  🙏

File: content/blog/2024-06-25-collaboration-networks/index.en.md [hunk 6]
```diff
@@ -0,0 +1,267 @@
...
+in order to **learn how to do it better** -- _Etienne Wenger_
+
+> A Community Manager **facilitates** the **activities** of a community and the **interactions** between **community members.** 
+Community management may be considered as _“in-reach”_ rather than _“outreach”_ or public engagement. - CSCCE
+
+rOpenSci is a community of practice and our passion is to build tools, resources, and social and technical infrastructure to enable open and reproducible research using R. We interact regularly in different activities and spaces in order to learn how to do it better.
+
+## Community Iteractions and Connectivity
+
+rOpenSci records a lot of data and generates statistics and summaries, for example, how many packages we reviewed, how many blog posts we wrote, how many community calls we organized and how many people came. This are very useful and show us an overal idea of our community and our activities.
```
[the remark this answers is not in the document]
Comment (yabellini) [reply content/blog/2024-06-25-collaboration-networks/index.en.md:41]: Yes! I like it more. I understand what you mean by the negative feeling!!

File: content/blog/2024-06-25-collaboration-networks/index.en.md [hunk 7]
```diff
@@ -0,0 +1,267 @@
...
+
+* for a blog post the nodes are the authors and the edge is build by authorship, by writing a post together. We can get that data from our website.  
+
+* For a package review, the nodes are the authors, editors and reviewers and the edge is the peer-review conducted together. The data source is our peer-review database and the github repo and issues where we conduct all the process.
+
+* The authors, mantainers and contributors of a package are nodes in the _Maintain a package_ way of contributions. The infomration is on GitHub.
+
+* Speak at a Community Call, the node are the speakers, host and moderators, and the edge is the participation in the same call. The data is in our community calls database and our webpage.
+
+* Become a champion, the nodes are the mentors and the mentees and the edge is the mentorship relationship. The data is in our mentorship program database and our webpage.
```
[the remark this answers is not in the document]
Comment (yabellini) [reply content/blog/2024-06-25-collaboration-networks/index.en.md:123]: Yes, good point. But that one is more complex to register.  When I can register, it is usually in another type of contribution, like writing something together, sharing an event (like a cohort meeting), or reviewing each other's package.

File: content/blog/2024-06-25-collaboration-networks/index.en.md (hunk 1 above)
Comment (yabellini) [reply content/blog/2024-06-25-collaboration-networks/index.en.md:249]: Thanks so much for the explanation on the difference of meaning on the words.

Comment (steffilazerte) [reply content/blog/2024-06-25-collaboration-networks/index.en.md]: My pleasure, it's fun for me to actually think about the nuances which are mostly instinctive. If I'm going to write it out, I have to make sure *I* understand 😉

File: content/blog/2024-06-25-collaboration-networks/index.en.md [hunk 8]
```diff
@@ -0,0 +1,267 @@
...
+Here is where Social Networks Analysis comes into play. I'm not talking about instagram or mastodon here, I'm talking about networks build by individuals u organization that have some kind of relationship.
+
+> Group of individuals who relate to others for a specific purpose, characterized by the existence of information flows.
+
+### Basic elements and metrics
+
+You can map the nodes and edges to explore the connections and patterns that exist
+and make conclusions based off of that exploration, for example, here we have map network, with the people as nodes and the edges as collaborations, for example, write a blog post together, being co-authors.
+
+{{< figure src = "red-elements.png" alt = "Two cartoon persons speaking with a can phone. Each person is a node, the thread is the edge and that edge can have direction.">}}
```
[the remark this answers is not in the document]
Comment (yabellini) [reply content/blog/2024-06-25-collaboration-networks/index.en.md]: Added the attribution and removed the direction arrow.

**Comment by yabellini** [comment]
@steffilazerte, I went through all your comments and suggestions and addressed them.  
Thanks so much for another great review. When I mention how awesome it is to work with the rOpenSci Team and how much I learn from my co-workers, this is an excellent example of that.
